src/core/qgsdatetimestatisticalsummary.cpp

```diff
@@ -66,6 +67,18 @@ void QgsDateTimeStatisticalSummary::addValue( const QVariant& value )
     testDateTime( date.isValid() ? QDateTime( date, QTime( 0, 0, 0 ) )
                   : QDateTime() );
   }
+  else if ( value.type() == QVariant::Time )
+  {
+    mIsTimes = true;
```

Comment:
Are we sure this method is only called once? Or does mIsTimes needs to be reset to false before this if-block?